be/src/exec/hdfs_scanner_orc.cpp

```diff
@@ -372,7 +385,46 @@ Status HdfsOrcScanner::do_open(RuntimeState* runtime_state) {
         _src_slot_descriptors.emplace_back(column.slot_desc);
         src_slot_index++;
     }
+    return Status::OK();
+}
 
+Status HdfsOrcScanner::do_open(RuntimeState* runtime_state) {
+    // create wrapped input stream.
+    RETURN_IF_ERROR(open_random_access_file());
+    if (_input_stream == nullptr) {
```

Comment:
When I put stream will be null?

Reply:
It's for testing.  by default it's null, but I'll set input stream in test suite.